### PR TITLE
Build for windows

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've rerendered the recipe as instructed in #6.

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
https://github.com/conan-io/conan-center-index/issues/8936

**Comment by hmaarrfk** [comment]
Seems like I am hitting the same bug as https://github.com/conda-forge/cmake-feedstock/pull/155#issuecomment-1037479664

**Comment by bmcfee** [comment]
:wave: I'm working on fixing up the libsndfile package, and was wondering what the status of this PR is.  I tried to view the failing build log, but it appears to have been purged already.

**Comment by hmaarrfk** [comment]
Fork and start?

**Comment by bmcfee** [comment]
Thanks!  Looks like the build is working (binary, dynamic and static libs generated), but the test fails because it's not copying lame.exe to the install path.

**Comment by hmaarrfk** [comment]
There are a few reasons for you to take it over the finish line if you can:

1. it seems like you know what you are doing.
2. it would make you maintainer for a package you care about
3. My time is quite limited :/ and sporadic.

File: conda-forge.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ build_platform: {osx_arm64: osx_64}
 conda_forge_output_validation: true
 provider: {linux_aarch64: azure, linux_ppc64le: azure}
 test_on_native_only: true
+build_with_mambabuild: false
```
Comment (hmaarrfk) [inline conda-forge.yml]:
```suggestion
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
   sha256: ddfe36cab873794038ae2c1210557ad34857a4b6bdc515785d1da9e175b1da1e
   patches:
     - libmp3lame.patch
+    - fixup_windows_build.patch
 
 build:
-  number: 1001
-  skip: True  # [win]
+  number: 1002
+  skip: true  # [not win]
```
Comment (hmaarrfk) [inline recipe/meta.yaml]:
```suggestion
```

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
I'm going to merge this, but I don't have any plans to test how this integrates into other software at conda forge. Please let me know if iyou have any issues with this and we can work together to resolve this.

**Comment by bmcfee** [comment]
> I'm going to merge this, but I don't have any plans to test how this integrates into other software at conda forge. Please let me know if iyou have any issues with this and we can work together to resolve this.

Thanks!  Once the mpg123 update gets merged, I'll resume the libsndfile update and report back if there are any issues.

> There are a few reasons for you to take it over the finish line if you can:
> 
>     1. it seems like you know what you are doing.
> 
>     2. it would make you maintainer for a package you care about
> 
>     3. My time is quite limited :/ and sporadic.


(1) might be a bit of an over-statement :grin: but I do have some motivation to get this working.  Re (2), my dependency chain looks like lame→libsndfile→pysoundfile→librosa (only the last of which I develop/maintain).  But keeping this distribution up to date and cross-platform will make my life easier with an upcoming deprecation cycle that I'm planning, so sure - happy to be looped in.  Re: (3) that's all of us I suppose!  I've been meaning to ping on this since about april, and only this weekend had time to dig into it.

Anyway, thanks for finishing this one off!

**Pull request merged**